src/common/routes/home/DsView.js

```diff
@@ -2478,12 +2478,14 @@ class DsView extends Component {
     }
 
     ajaxResponse (url, params, response) {
-        //console.log(`In ajaxResponse, params: ${JSON.stringify(params, null, 4)}, url: ${url},  response.total: ${response.total}, response.reqCount: ${response.reqCount}, state.initialHeaderFilter: ${JSON.stringify(this.state.initialHeaderFilter, null, 4)}`);
-        if ((response.reqCount == this.reqCount) || (response.reqCount == 0)) {
-            this.setState({ totalRecs: response.total});
-        } else {
-            console.log(`In ajaxResponse, avoided stale setting of response.total!`);
-        }
+        // console.log(`In ajaxResponse, params: ${JSON.stringify(params, null, 4)}, url: ${url},  response.total: ${response.total}, response.reqCount: ${response.reqCount}, state.initialHeaderFilter: ${JSON.stringify(this.state.initialHeaderFilter, null, 4)}`);
+        setTimeout(() => {
```

Comment:
I don't think this is the right place for adding timeout. Try reading the documentation of header filter on https://tabulator.info/docs/5.5/filter#func-builtin  and find the function which is triggered when we enter some text in the header filter of some column. We have to add some kind of debounce timer on that function.